components/base/kinematicbase/differentialDrive.go

```diff
@@ -54,13 +54,21 @@ func wrapWithDifferentialDriveKinematics(
 	if len(geometries) > 0 {
 		geometry = geometries[0]
 	}
-	ddk.executionFrame, err = referenceframe.New2DMobileModelFrame(b.Name().ShortName(), limits, geometry)
+	sphere, err := spatialmath.BoundingSphere(geometry)
+	if err != nil {
+		// default to using a point to represent the base for the purposes of collision handling rather than erroring
+		logger.Warnf("base %s not configured with a geometry, defaulting to considering it a point mass." +
```

Comment:
Pretty long, any way to make it snappier?

Reply:
You're right I will try and channel Hemingway on the rewrite